packages/plugin-ext/src/plugin/tree/tree-views.ts

```diff
@@ -247,11 +247,14 @@ class TreeViewExtImpl<T> implements Disposable {
 
     async reveal(element: T, options?: Partial<TreeViewRevealOptions>): Promise<void> {
         await this.pendingRefresh;
+        const select = !(options?.select === false); // default to true
```

Comment:
suggestion:

```suggestion
        const select = options?.select !== false; // default to true
```

`true` → `true`
`false` → `false`
`undefined` → `true`